URL_LINKS_REFERENCE.md

```
@@ -0,0 +1,463 @@
+|------|-----|-------------|
+| E-Invoice | `https://yourdomain.com/einvoice` | E-invoice generation |
+| Reports | `https://yourdomain.com/reports` | Analytics & reports |
+
+### **System**
+| Page | URL | Description |
+|------|-----|-------------|
+| Notifications | `https://yourdomain.com/notifications` | User notifications |
+| Settings | `https://yourdomain.com/settings` | Business settings |
+
+### **Subscription & Billing** ⭐
+| Page | URL | Description |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)